### PR TITLE
Adds new monster balls to call Kevin your pokemon I mean pocket monster and bully with

### PR DESCRIPTION
**Pull request by Xantholne**
## About The Pull Request

Adds new monster balls, a more expensive bluespace jar that does not break when thrown. Also some sick sci fi sounds to fit them.

## Why It's Good For The Game

Allows miners, xenobilogists, people who bully kevin, etc, to have a reliable means to pick up monsters, or catboys, and throw them around like the pocket monster they are. Now you too can be a pokemon master with your 10k mining points to blow on lazarus injectors for the goliath fetish army you want.

## Changelog
:cl:
add: New monster balls as a more expensive bluespace jar.
/:cl:

Vx99GVkRA4
dreamseeker_nOLdhvf84I
dreamseeker_83Nxwxa6OS

### CODE REVIEW

**Comment by Detective-Google** [comment]
`[Nintendo of America would like to know your location]`

**Comment by keronshb** [comment]
Can these pick up players?

If so they should have a hardstun like the Jar is going to have to prevent dumb shit.

**Comment by MacaroniCritter** [comment]
I fucking hate pokemon but like xantholne +1

File: code/game/objects/items/pet_carrier.dm [hunk 1]
```diff
@@ -350,4 +350,140 @@
...
+	///level until the reagent gets INGEST ed instead of TOUCH
+	var/sipping_level = 150
+	///prob50 level of sipping
+	var/sipping_probably = 99
+	///chem transfer rate / second
+	var/transfer_rate = 5
+
+/obj/item/pet_carrier/bcball/Initialize()
+	. = ..()
+	create_reagents(300, OPENCONTAINER, DEFAULT_REAGENTS_VALUE) //equivalent of bsbeakers
```
Comment (LetterN) [inline code/game/objects/items/pet_carrier.dm:388]: I dont think we need the reagent part?

**Comment by Xantholne** [comment]
> Can these pick up players?
> 
> If so they should have a hardstun like the Jar is going to have to prevent dumb shit.

Yes, same as bluespace jars.

File: code/game/objects/items/pet_carrier.dm (hunk 1 above)
Comment (Xantholne) [reply code/game/objects/items/pet_carrier.dm:388]: Thought about it, mixed on keeping or not. Haven't fully decided but I'm leaning towards removing adding reagents

**Comment by YakumoChen** [comment]
Port the antag miner belt with random mobs

File: code/game/objects/items/pet_carrier.dm [hunk 2]
```diff
@@ -350,4 +350,140 @@
...
+/obj/item/pet_carrier/bcball/update_icon_state()
+	if(open)
+		icon_state = "bcball_open"
+	else
+		icon_state = "bcball"
+
+/obj/item/pet_carrier/bcball/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..()
+	if(occupants.len)
+		loc.visible_message("<span class='warning'>I choose you,[occupants[1]]!</span>")
```
Comment (LetterN) [inline code/game/objects/items/pet_carrier.dm]:
```suggestion
		loc.visible_message("<span class='warning'>I choose you, [occupants[1]]!</span>")
```

File: code/game/objects/items/pet_carrier.dm [hunk 3]
```diff
@@ -350,4 +350,140 @@
...
+			var/turf/AT = get_turf(thrownby)
+			throwerstring = " THROWN BY [key_name(thrownby)] at [AT] (AREACOORD(AT)]"
+		log_reagent("SPLASH: [src] mob throw_impact() onto [key_name(hit_atom)] at [TT] ([AREACOORD(TT)])[throwerstring] - [R]")
+		reagents.reaction(hit_atom, TOUCH)
+		reagents.clear_reagents()
+
+	if(iscatperson(occupants[1]))
+		var/mob/living/L = occupants[1]
+		// here's a big L for you
+		L.emote("meow")
```
Comment (LetterN) [inline code/game/objects/items/pet_carrier.dm:434]:
```suggestion
	var/mob/living/L = occupants[1]
	// here's a big L for you
	if(istype(L) && iscatperson(L))
		L.emote("nya")
```

**Review by LetterN — CHANGES_REQUESTED** [review]

**Comment by TheEaterOfBeans** [comment]
the smelly ipc

**Comment by DeltaFire15** [comment]
While a pretty nice concept, we really don't need a second bluespace jar to have to balance in tandem, with the same issues, if not more (considering these are intended to be thrown repeatedly and reused)
Silicons seemed to have the same opinion on this, so, closing this PR.

**Pull request closed without merging**